src/pages/stats/[login].tsx

```diff
@@ -126,6 +127,26 @@ export default function Stats() {
     return text;
   }
 
+  const copyToClipboard = (format: "text" | "json") => {
+    let data = null;
+
+    if (format === "text") data = generateText();
+    else data = JSON.stringify(repositories, null, 2);
+
+    format = format.charAt(0).toUpperCase() + format.slice(1);
```

Comment:
The build fails because of this assignment, format should be `"text" | "json"` but here you assign a string.

I'd probably remove this line entirely as it's not necessary, we can keep the lowercase string for the toasts.

Reply:
I will fix it

Reply:
I have push new code, help me check it